tests/unit/utils/test_fs.py

```diff
@@ -149,3 +150,10 @@ def test_relpath_windows_different_drives():
     rel_info = relpath(info1, info2)
     assert isinstance(rel_info, str)
     assert rel_info == path1
+
+
+def test_get_inode(repo_dir, dvc_repo):
+    # Mocking a file to get the inode.
+    path = os.path.join(repo_dir.FOO)
```

Comment:
Why are you joining a single file? :)
```suggestion
    path = repo_dir.FOO
```

Reply:
@efiop I thought I could create a valid path this way only. :sweat_smile: